runner/runners/README.md

```diff
@@ -0,0 +1,31 @@
+# Runner Impl Design #
+package runner defines Runner, which lets clients start, stop, and get the status of Scoot Commands. This package contains implementations of Runner.
+
+There are three main components of our Runner Impl:
+* Statuses
```

Comment:
The relationship between these is confusing. Not clear which are interfaces (controller, statuser), implementations (queuingcontroller, statuses), and just structs (invoker).  Seems like at a high level, that status and controller are most important, and invoker is an implementation detail of our particular controller.